src/event.rs

```diff
@@ -1081,16 +1098,24 @@ impl InnerSizeWriter {
         Self { new_inner_size }
     }
 
-    /// Try to request inner size which will be set synchroniously on the window.
+    /// Try to request a new inner size which will be set synchronously on the
+    /// window.
+    ///
+    ///
+    /// # Errors
+    ///
+    /// This method returns an error when the request to change the inner size
+    /// of the window was ignored - likely because you tried to change the
```

Comment:
I don't think "likely" is necessary, I believe this is the only way this can fail.